x-pack/plugins/security_solution/public/resolver/view/panels/related_event_detail.tsx

```diff
@@ -23,6 +23,19 @@ import * as selectors from '../../store/selectors';
 import { useResolverDispatch } from '../use_resolver_dispatch';
 import { PanelContentError } from './panel_content_error';
 
+/**
```

Comment:
:grey_question: It's really only for display purposes, but should we move it to a selector?